nmostesting/suites/IS1101Test.py

```diff
@@ -0,0 +1,3813 @@
+                    return test.FAIL(
+                        "The streamcompatibility request for sender {} status has failed: {}".format(
+                            sender_id, response
+                        )
+                    )
+
+                try:
+                    flow_id = response.json()["flow_id"]
+                except json.JSONDecodeError:
+                    return test.FAIL("Non-JSON response returned from Node API")
+                except KeyError as e:
+                    return test.FAIL("Unable to find expected key: {}".format(e))
+
+                if flow_id is None:
+                    return test.FAIL("flow_id is null")
+                valid, response = TestHelper.do_request(
+                    "GET", self.node_url + "flows/" + flow_id
+                )
+                if not valid:
+                    return test.FAIL(
+                        "Unexpected response from the streamcompatibility API: {}".format(
+                            response
+                        )
+                    )
+                if response.status_code != 200:
+                    return test.FAIL(
+                        "The streamcompatibility request for sender {} status has failed: {}".format(
+                            sender_id, response
+                        )
+                    )
+                try:
+                    grain_rate = response.json()["grain_rate"]
+                except json.JSONDecodeError:
+                    return test.FAIL("Non-JSON response returned from Node API")
+                except KeyError as e:
+                    return test.FAIL("Unable to find expected key: {}".format(e))
+
+                if grain_rate != self.get_another_grain_rate(
+                    self.flow_grain_rate[sender_id]
+                ):
+                    return test.FAIL(
+                        "The flow_grain_rate does not match the constraint"
+                    )
+                valid, response = TestHelper.do_request(
+                    "DELETE",
+                    self.compat_url + "senders/" + sender_id + "/constraints/active/",
+                )
+                if not valid:
+                    return test.FAIL(
+                        "Unexpected response from the streamcompatibility API: {}".format(
+                            response
+                        )
+                    )
+                if response.status_code != 200:
+                    return test.FAIL(
+                        "The streamcompatibility request for sender {} status has failed: {}".format(
+                            sender_id, response
+                        )
+                    )
+            return test.PASS()
+        return test.UNCLEAR("No resources found to perform this test.")
+
+    def test_02_03_05_02(self, test):
+        """
+        Verify for inputs supporting EDID that the version and the effective EDID change when applying constraints
+        """
```

Comment:
This has the same test description as test_02_03_05_01. What's different?

Reply:
Maybe name one with video and the other one with audio

Reply:
2.3.5.1 is video, 2.3.5.2 is audio

Reply:
```suggestion
        """
        Verify for audio inputs supporting EDID that the version and the effective EDID change when applying constraints
        """
```